### PR TITLE
Update installation documentation

### PR DESCRIPTION
**Pull request by dnaka91**
I recently created a package for Arch Linux (AUR), that allows to easily build and install MemeBox on that specific Linux OS. Thought, it's worth mentioning that in the readme.

In addition, I added instructions on how to build the whole application (both headless and Electron variant) from source, for anybody who wants to build it themselves instead of using pre-compiled binaries.

### CODE REVIEW

**Comment by negue** [comment]
WOW - That is awesome!! It explains it very good 🎉 

With this amount of information, maybe it would be better to split this into its own markdown file, what do you think ?

Dunno whats the norm - maybe an `installation.md` or something and then link to this

**Comment by negue** [comment]
maybe this even needs to extracted from https://github.com/meme-box/memebox-101/blob/main/00/README.md as well

I can do that afterwards so that its all atleast in one place 😁

**Comment by dnaka91** [comment]
Oh, didn't know about memebox-101. That seems more like an end-user guide. Personally, wouldn't say that build instructions are that helpful for the average non-dev user. But can't hurt to add it there too, with some nice screenshots and more visual aid.

There doesn't really seem to be a clear best practice on where to put the build instructions. At least from what I can tell. Some project keep literally everything in a single README.md (not a big fan of that), some have all the docs in a separate web page, and others have only "from source" instructions in a separate INSTALLATION.md or BUILD.md.

I guess it highly depends on the maturity and scale of the project. Bigger projects tend to have their own site at some point. And even there I see the difference between keeping docs in a sub-folder or separate repo.

---

Long story short, I'd say it's good to keep the basic installation instructions in the main README.md, like the "From AUR" section, which would likely be expanded to "From Homebrew", "From Chocolatey" and others. They don't really bloat much, as each section is just one sentence and an optional shell command sample.

The for the "From source" section, we move the details into a separate file and just reference it in a single sentence like: 
> For instructions of how to build MemeBox from source, have a look at the X.md.

Also, maybe BUILD.md is more fitting if we only extract that part into the file. As the main installation instructions are kept in the main README.md.

What do you think?

**Comment by negue** [comment]
This is great!! 

> Oh, didn't know about memebox-101. That seems more like an end-user guide. Personally, wouldn't say that build instructions are that helpful for the average non-dev user. But can't hurt to add it there too, with some nice screenshots and more visual aid.

Yeah you are right, it is more for an end-user not a developer, and its probably the best to keep those separately.

> Also, maybe BUILD.md is more fitting if we only extract that part into the file. As the main installation instructions are kept in the main README.md.

Yes BUILD.md does sound like the best way for that - I saw that one in a different repo as well so - lets go with that (and you already did :+1: )

I might combine those installation sources into a table but I don't know what kind of layout it should be so this can be done some other time - for now - this is noice and now we also have a good instruction on how to build it

Thank you!

**Review by negue — APPROVED** [review]
